mmcv/utils/__init__.py

```diff
@@ -17,6 +17,7 @@
                       check_python_script)
 from .timer import Timer, TimerError, check_time
 from .version_utils import digit_version, get_git_hash
+from .ipu_wrapper import IPU_MODE
```

Comment:
suggest renaming `IPU_MODE` to `is_ipu_pytorch` which is consistent with `is_rocm_pytorch`.